openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1241,7 +1241,7 @@ Authorization: Bearer czZCaGRSa3F0MzpnWDFmQmF0M2JW
 {
   "notification_id": "3fwe98js",
   "event": "credential_deleted",
-  "error_description": "..."
+  "event_description": "..."
```

Comment:
Either input some text her or omit the optional parameter

Reply:
I think having an example is good; perhaps:

```suggestion
  "event_description": "User pressed 'cancel' button"
```

Reply:
does cancel button match with the event "credential_deleted" ? Re-reading the options for event, "credential_deleted" was confusing to me.

Reply:
If I'm correct, this is only about the acknowledgement of the credential response? So isn't the credential denied, refused or rejected instead of deleted?

Reply:
It's pretty much the only thing that can trigger 'deleted' I think, the spec defines deleted as "to be used when the unsuccessful Credential issuance was caused by a user action".

Reply:
> If I'm correct, this is only about the acknowledgement of the credential response?

Yes, as the spec is currently written.

> So isn't the credential denied, refused or rejected instead of deleted?

I would suggest not reopening that argument in this PR :-)